### PR TITLE
Restored advanced who functionality for admins

### PR DESCRIPTION
**Pull request by optimumtact**
This new verb will be called awho, do what who did previously and be logged.

### CODE REVIEW

**Comment by Supermichael777** [comment]
Rude

**Comment by optimumtact** [comment]
Start shit get hit

**Comment by phil235** [comment]
Change the name of the verb...

**Comment by optimumtact** [comment]
okay

File: code/game/verbs/who.dm [hunk 1]
```diff
@@ -26,6 +26,46 @@
...
+				var/entry = "\t[C.key]"
+				if(C.holder && C.holder.fakekey)
+					entry += " <i>(as [C.holder.fakekey])</i>"
+				Lines += entry
+		for(var/line in sortList(Lines))
+		msg += "[line]\n"
+		msg += "<b>Total Players: [length(Lines)]</b>"
+	
+	else
+		msg = "<b>Kor Aint shit</b>"
```
Comment (Aranclanos) [inline code/game/verbs/who.dm]: ?

Comment (RemieRichards) [inline code/game/verbs/who.dm]: the entire proc is called ```koraintshitwho()```, it's <i>all memes</i>

**Comment by Aranclanos** [comment]
Put a brief description of what are you doing in the OP

**Comment by Jalleo** [comment]
This is a poor joke and you should feel bad

**Comment by optimumtact** [comment]
It was a meme at first but this will be an advanced who for admins likely called awho

**Comment by Fayrik** [comment]
You know, I was going to readd this to the regular who command but only have it function if the admin was also a ghost.
And then it'd only log like once per round.

...Yeah, that was gunna be a lot of work.

File: code/game/verbs/who.dm (hunk 1 above)
Comment (optimumtact) [inline code/game/verbs/who.dm]: rude this is a useful admin feature

**Comment by Aranclanos** [comment]
no memes please

**Comment by optimumtact** [comment]
This isn't a meme I just haven't finished

**Comment by Aranclanos** [comment]
then why do you commit faggot

**Comment by optimumtact** [comment]
For feedback, something you know nothing of

**Comment by Aranclanos** [comment]
feed? im not hungry

**Pull request closed without merging**